src/main/java/com/minecolonies/coremod/placementhandlers/PlacementHandlers.java

```diff
@@ -1,6 +1,7 @@
 package com.minecolonies.coremod.placementhandlers;
 
 import com.minecolonies.coremod.blocks.BlockSolidSubstitution;
+import com.minecolonies.coremod.colony.jobs.AbstractJob;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.colony.jobs.AbstractJob'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)